.github/workflows/main.yml

```
@@ -57,16 +98,59 @@ jobs:
+        env:
+          ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
+        run: |
+          make generate-notes VERSION_OLD=${{ needs.update_version.outputs.previous_version }} VERSION_NEW=${{ github.ref_name }} > release_notes.md
 
-      - name: Upload Release Asset
+      - name: Update Release Notes
         uses: softprops/action-gh-release@v2
         with:
-          files: kodi/plugin.video.mediafusion-*.zip
+          body_path: release_notes.md
+          token: ${{ secrets.GITHUB_TOKEN }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)